libs/features/webapp-prompt/tooltip.css

```diff
@@ -0,0 +1,50 @@
+[data-pep-dismissal-tooltip]::after {
+  content: attr(data-pep-dismissal-tooltip);
+  position: absolute;
+  display: inline-flex;
+  z-index: 201;
```

Comment:
Is this high value actually needed? We usually go for the lowest value that would achieve the desired result. Is Unav setting some values to justify this?

Reply:
I've changed the z-indices to 3 and 2 so they remain higher than the z-index of the hover tooltip.